app/src/main/res/layout/fragment_buffer.xml

```diff
@@ -0,0 +1,24 @@
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.fragment_buffer` appears to be unused
 Unused resources make applications larger and slow down builds.